### ISSUE TITLE
argparse Target Handling

### ISSUE THREAD

**Issue by DarthDobber** [leapsecurity/libssh-scanner#4]
The code for the target parsing is incomplete.  You need to add '-t' and '--target'

**Comment by jobroche** [comment]
try `./libsshscan.py <target>`

I removed the need for -t or --target with yesterday's update

**Comment by ximad** [comment]
the README need modifications ;) and a -out for long ip range can be cool

**Comment by jobroche** [comment]
@ximad -out to save to a file?

**Comment by ximad** [comment]
yeah , something like the -oA of nmap could be better , but yes start with a txt file for exemple ( i have put 659 ip in a txt target , and .... it's a pain ;) ) . many thx

**Comment by jobroche** [comment]
Ok, this will be included in the next release. For now try `./libssh <targets> | tee output.txt`

**Comment by prakhar-mishra-hub** [comment]
Any exploit for cve-2018-15919???
